### ISSUE TITLE
E305 fixer not satisfying flake8

### ISSUE THREAD

**Issue by peterjc** [hhatto/autopep8#304]
Short test case:

```
$ cat example.py 
import os


class Silly(object):

    def __call__(self):
        return os.sep

if __name__ == "__main__":
    silly = Silly()
    print(silly())
```

The current version of pep8 is happy:

```
$ pep8 --version ; pep8 --select E305 example.py 
1.7.0
```

The current version of flake8 is stricter:

```$ flake8 --version ; flake8 --select E305 example.py 
3.3.0 (mccabe: 0.6.1, pycodestyle: 2.3.1, pyflakes: 1.5.0) CPython 3.6.0 on Darwin
example.py:9:1: E305 expected 2 blank lines after class or function definition, found 1
```

The current version of autopep8 does not make the change wanted by flake8:

```
$ autopep8 --version ; autopep8 --in-place --select E305 example.py 
autopep8 1.2.4
$ flake8 --version ; flake8 --select E305 example.py 
3.3.0 (mccabe: 0.6.1, pycodestyle: 2.3.1, pyflakes: 1.5.0) CPython 3.6.0 on Darwin
example.py:9:1: E305 expected 2 blank lines after class or function definition, found 1
```

Desired change is to insert another blank line,

```
$ cat example2.py 
import os


class Silly(object):

    def __call__(self):
        return os.sep


if __name__ == "__main__":
    silly = Silly()
    print(silly())
$ diff example.py example2.py 
8a9
> 
$ flake8 --version ; flake8 --select E305 example2.py 
3.3.0 (mccabe: 0.6.1, pycodestyle: 2.3.1, pyflakes: 1.5.0) CPython 3.6.0 on Darwin
```

**Comment by hhatto** [comment]
fix in latest master.

It is scheduled to be released in version 1.3. (next week)

**Comment by peterjc** [comment]
Thanks
